pkg/utils/utils.go

```diff
@@ -182,6 +210,27 @@ func SyncNodeState(newState *sriovnetworkv1.SriovNetworkNodeState, pfsToConfig m
 				continue
 			}
 
+			// load the PF info
+			pfStatus, exist, err := LoadPfsStatus(ifaceStatus.PciAddress, false)
+			if err != nil {
+				glog.Errorf("SyncNodeState(): failed to load info about PF status for pci address %s: %v", ifaceStatus.PciAddress, err)
+				return err
+			}
+
+			if !exist {
```

Comment:
if entry was not found, does this nessecarily mean that VFs were not created by sriov-operator ?